cmd/ci-operator-configresolver/main.go

```diff
@@ -0,0 +1,160 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/ghodss/yaml"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/interrupts"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/config"
+)
+
+const (
+	orgQuery     = "org"
+	repoQuery    = "repo"
+	branchQuery  = "branch"
+	variantQuery = "variant"
+)
+
+type filenameToConfig map[string]api.ReleaseBuildConfiguration
```

Comment:
(we don't have the `org/repo/` subdir structure and thereby cannot use the normal loaders, or generate normal `Info` objects. That's a shame.)

Reply:
We can't build a correct `config.Info` struct since we only have the filename and not the full directory structure, so `config.ByFilename` doesn't work for this (unless we want to intentionally omit the `Info` struct; we would still need to have a separate function to load the configs because the `LoadConfigByFilename` function sets the filename using the directory structure (because it sets the basename based on the `Info` struct, which is generated by `InfoFromPath`)).